src/app/core/models/customer/customer.mapper.ts

```diff
@@ -48,8 +48,15 @@ export class CustomerMapper {
         }
       : {
           customerNo: data.customerNo,
-          type: data.type,
+          type: 'PrivateCustomer',
           isBusinessCustomer: false,
         };
   }
+
+  private static isBusinessCustomer(data: CustomerData): boolean {
+    if (data.type === 'PrivateCustomer') {
+      return false;
+    }
+    return true;
+  }
```

Comment:
Now would be the perfect time to solve this once and for all: https://github.com/intershop/intershop-pwa/projects/4#card-35562084

Only use the 'isBusinessCustomer' field and remove the 'type' field and also remove this helper method, please :wink:

Reply:
done in develop